functions.js

```diff
@@ -0,0 +1,52 @@
+const listContainer = document.querySelector('.list-container');
+const itemsContainer = document.querySelector('.list-container');
+
+export default class Collection {
+  static tasks = [];
+
+  static createTask = (task) => {
+    const objTasks = {
+      description: task,
+      completed: false,
+      index: Collection.tasks.length + 1,
```

Comment:
You should refer to the fields and methods by using the "this" javascript keyword. It is a best practice followed by many, It helps your code to be more consistent.